src/gwt/acesupport/acemode/utils.js

```diff
@@ -278,7 +278,7 @@ var YamlHighlightRules = require("mode/yaml_highlight_rules").YamlHighlightRules
          if (state === "start" || state.indexOf("-start") !== -1) {
             self.$rules[state].unshift({
                token: "comment.doc.tag",
-               regex: "^\\s*#\\s*[|]",
+               regex: "^\\s*#[|]",
```

Comment:
```suggestion
               regex: "^\\s{0,10}#[|]",
```
Should there be a maximum?

I wouldn't see why someone would expect this to highlight quarto comments?

```
                             #|
```

Reply:
It could matter for indented chunks; apparently those are processed by Quarto. The only requirement appears to be that the indentation level for the comment prefix matches the chunk indentation. Supporting that here is more challenging.

We can consider that in a future change if necessary.